space2stats_api/src/pyproject.toml

```diff
@@ -21,7 +21,8 @@ starlette-cramjam = ">=0.3,<0.4"
 mangum = "*"
 asgi-s3-response-middleware = "^0.0.4"
 boto3 = "^1.35.11"
-numpy = "^2.1.1"
+numpy = "^1.24.0"
```

Comment:
Hmm we should get h3ronpy updated to work with numpy v2. That depends on this PR https://github.com/PyO3/rust-numpy/pull/442

Reply:
While waiting for this to be resolved: https://github.com/nmandery/h3ronpy/issues/58, I opted to use the cells_to_coordinates(h3_ids_uint64) function.